semgrep-core/src/reporting/JSON_report.ml

```diff
@@ -101,8 +101,39 @@ let range_of_any_opt startp_of_match_range any =
       let startp, endp = position_range min_loc max_loc in
       Some (startp, endp)
 
+let metavar_string_of_any any =
+  any |> V.ii_of_any
+  |> List.filter PI.is_origintok
+  |> List.sort Parse_info.compare_pos
+  |> Common.map PI.str_of_info |> Matching_report.join_with_space_if_needed
+
+let get_propagated_value default_start metavar =
+  let any_to_svalue_value any =
+    match range_of_any_opt default_start any with
+    | Some (start, end_) ->
+        Some
+          {
+            ST.svalue_start = start;
+            svalue_end = end_;
+            svalue_abstract_content = metavar_string_of_any any;
+          }
+    | None -> None
+  in
+  match metavar with
+  | E { e = N (Id (_, id_info)); _ } -> (
```

Comment:
Is `metavar` here really a metavariable? To my knowledge only code variables will have a relevant `id_svalue`.

Reply:
Maybe rename it to `mvalue`?